.npmrc

```
@@ -0,0 +1,3 @@
+//verdaccio.faire.team/:_authToken=${VERDACCIO_TOKEN}
+//verdaccio.faire.team/:always-auth=true
+@faire:registry=https://verdaccio.faire.team/
```

```suggestion
//verdaccio.faire.team/:_authToken=${VERDACCIO_TOKEN}
//verdaccio.faire.team/:always-auth=true
registry=https://verdaccio.faire.team/
```

nit: might cause behavioural issues in local vs CI